Add send data as note in install instructions (#1225)
timescale/docs: promscale/installation/docker.md

</procedure>

> <highlight type="note">
> Post installation of Promscale, the next step is to ingest the 
> data into Promscale. Follow the [send data][send-data] instructions.
> </highlight>

Reviewer: ```suggestion
When you have installed Promscale, you are ready to ingest 
data. For instructions, see the [send data][send-data] section.
```

Reviewer: Just an additional note: reference-style links won't work inside an admonition. I haven't changed that, though, since I took it out of the admonition.

</procedure>

> When you have installed Promscale, you are ready to ingest 
> data. For instructions, see the [send data][send-data] section.

Reviewer 2: ```suggestion
After you have installed Promscale, you can ingest data.
For instructions, see the [send data][send-data] section.
```
